README.md

```diff
@@ -1258,6 +1259,54 @@ const client = new GraphQLClient({
 })
 ```
 
+### abortController
+
+if you wish to abort a fetch it is possible to pass an abort controller signal to the `fetchOptionsOverrides` option of the fetch function. This is not `graphql-hooks` specific functionality, rather just an example of how to use it with the library.
+
+```js
+import { GraphQLClient, useManualQuery } from 'graphql-hooks'
+
+const client = new GraphQLClient({
+  url: '/graphql'
+})
+
+const controller
+
+function App() {
+  const [fetchPosts, { loading, error }] = useManualQuery(
+    `
+      query {
+        allPosts {
+          id
+          title
+          body
+        }
+      }
+    `
+  )
+
+  const handleFetch = () => {
+    controller = new AbortController()
+    await fetchPosts({
+      fetchOptionsOverrides: {
+        signal: controller.signal
+      }
+    })
+  }
+
+  const handleAbort = () => {
+    controller.abort()
+  }
```

Comment:
minor: storing values that a React component uses inside module variables is not a very common practice, as you have state lingering around outside of the scope of the component's lifecycle. I understand that this is a simplified example, but since the _correct_ version (which uses Refs) isn't considerably more complex to put in place, I'd go that way to avoid showing misleading code samples